.github/workflows/release.yml

```diff
@@ -127,22 +127,28 @@ jobs:
 
```

Comment:
The code change looks correct. It handles the case where the `context.payload.pull_request` is null by checking if `pr_number` is truthy before making the API call. This is a good defensive coding practice.